cmd/solana_exporter/exporter.go

```diff
@@ -14,22 +14,28 @@ import (
 )
 
 var (
-	httpTimeout      = 60 * time.Second
-	rpcAddr          = flag.String("rpcURI", "", "Solana RPC URI (including protocol and path)")
-	addr             = flag.String("addr", ":8080", "Listen address")
-	votePubkey       = flag.String("votepubkey", "", "Validator vote address (will only return results of this address)")
-	httpTimeoutSecs  = flag.Int("http_timeout", 60, "HTTP timeout in seconds")
-	balanceAddresses = flag.String("balance-addresses", "", "Comma-separated list of addresses to monitor balances")
+	httpTimeout         = 60 * time.Second
+	rpcAddr             = flag.String("rpcURI", "", "Solana RPC URI (including protocol and path)")
+	addr                = flag.String("addr", ":8080", "Listen address")
+	votePubkey          = flag.String("votepubkey", "", "Validator vote address (will only return results of this address)")
+	httpTimeoutSecs     = flag.Int("http_timeout", 60, "HTTP timeout in seconds")
+	balanceAddresses    = flag.String("balance-addresses", "", "Comma-separated list of addresses to monitor balances")
+	leaderSlotAddresses = flag.String(
+		"leader-slot-addresses",
+		"",
+		"Comma-separated list of addresses to monitor leader slots by epoch for, leave nil to track by epoch for all validators.",
```

Comment:
Might want to add a warning here that it will have a very large number of metrics when doing this. Actually, in this case perhaps just log a warning at start.

Reply:
Good idea, done.